src/EmojiBlock.tsx

```diff
@@ -21,16 +21,14 @@ export default function EmojiBlock({
     }
   }, [state]);
 
-  const mappedReactions = state
-    .sort((a, b) => (a.emoji < b.emoji ? 1 : -1))
```

Comment:
Unfortunately the sort is necessary, otherwise on each click the emojis would change position.

Reply:
sorry! Wasn't gonna delete that was just testing.